bigcommerce/customer_login_token.py

```diff
@@ -0,0 +1,55 @@
+import os
+import time
+import uuid
+import jwt
+
+# Get the client_secret needed to sign tokens from the environment
+# Intended to play nice with the Python Hello World sample app
+# https://github.com/bigcommerce/hello-world-app-python-flask
+client_secret = os.getenv('APP_CLIENT_SECRET', None)
+
+
+class CustomerLoginTokens:
+    @classmethod
+    def create(cls, client, id, redirect_url=None, request_ip=None):
+
+        if not client_secret:
+            raise AttributeError('No OAuth client secret specified in the environment, '
+                                 'please specify an APP_CLIENT_SECRET')
+
+        try:
+            client_id = client.connection.client_id
+        except AttributeError:
+            raise AttributeError('No client ID found in the connection - '
+                                 'make sure an OAuth API connection is configured')
+
+        try:
+            store_hash = client.connection.store_hash
+        except AttributeError:
+            raise AttributeError('No store hash found in the connection - '
+                                 'make sure an OAuth API connection is configured')
```

Comment:
same as above. If the fix is the same, the two statements inside the try: can be merged into one try block.

🍸